docs/aserto-console/directory/add-relation.mdx

```diff
@@ -0,0 +1,68 @@
+---
+sidebar_label: Adding Relations
+title: Adding Relations
+description: Adding relations between subjects and objects 
+---
+
+# Adding Relations
+
+Now that we have our Organization object type, the Viewer, Editor, and Owner relation types, and the associated permissions, we can assign users into the various relations.
+
+Let's assign Rick the "owner" relation to the Citadel organization, and the "viewer" relation to the Smith organization.
```

Comment:
we could add a step for this - `"viewer" relation to the Smith organization.` or a "similarly, ..."

Reply:
reworked this